pkg/sql/sqlliveness/slstorage/slstorage.go

```diff
@@ -247,29 +247,13 @@ func (s *Storage) isAlive(
 }
 
 func (s *Storage) getReadCodec(version *settingswatcher.VersionGuard) keyCodec {
-	if version.IsActive(clusterversion.TODO_Delete_V23_1_SystemRbrReadNew) {
-		return s.newKeyCodec
-	}
-	return s.oldKeyCodec
-}
-
-func (s *Storage) getDualWriteCodec(version *settingswatcher.VersionGuard) keyCodec {
-	switch {
-	case version.IsActive(clusterversion.TODO_Delete_V23_1_SystemRbrSingleWrite):
-		return nil
-	case version.IsActive(clusterversion.TODO_Delete_V23_1_SystemRbrReadNew):
-		return s.oldKeyCodec
-	case version.IsActive(clusterversion.TODO_Delete_V23_1_SystemRbrDualWrite):
-		return s.newKeyCodec
-	default:
-		return nil
-	}
+	return s.newKeyCodec
 }
 
 func (s *Storage) versionGuard(
```

Comment:
nit: The versionGuard helpers in slstorage, sqlinstance, and lease can be deleted. I see you added a TODO to the lease helper, but not the slstorage or sqlinstance helpers.